### DISCUSSION TITLE
Prior preservation

### DISCUSSION THREAD

**Discussion by abc123desygn** [TheLastBen/fast-stable-diffusion#1782] [Q&A]
I notice people complaining about overfitting and settings, is there any particular reason why the prior preservation flag isn't used with the dreambooth training script e.g  '**--with_prior_preservation --prior_loss_weight=1.0**' and the notebook discourages the use of regularization when training faces? 
I'm referring to this article from Huggingface https://huggingface.co/blog/dreambooth

**Comment by JustAnOkapi** [comment]
its dirty

**Comment by TheLastBen** [comment]
Early stopping the text encoder training is a better regularization method
